libmscore/note.cpp

```diff
@@ -106,20 +212,74 @@ struct NoteHeadName {
 
 // same order as NoteHead::Group
 static NoteHeadName noteHeadGroupNames[] = {
-      {"normal",    QT_TRANSLATE_NOOP("noteheadnames", "Normal") },
-      {"cross",     QT_TRANSLATE_NOOP("noteheadnames", "Cross") },
-      {"diamond",   QT_TRANSLATE_NOOP("noteheadnames", "Diamond") },
-      {"triangle",  QT_TRANSLATE_NOOP("noteheadnames", "Triangle") },
-      {"mi",        QT_TRANSLATE_NOOP("noteheadnames", "Mi") },
+      {"normal",         QT_TRANSLATE_NOOP("noteheadnames", "Normal") },
+      {"cross",          QT_TRANSLATE_NOOP("noteheadnames", "Cross") },
+      {"plus",           QT_TRANSLATE_NOOP("noteheadnames", "Plus") },
+      {"xcircle",        QT_TRANSLATE_NOOP("noteheadnames", "XCircle") },
+      {"withx",          QT_TRANSLATE_NOOP("noteheadnames", "With X") },
+      {"triangle-up",    QT_TRANSLATE_NOOP("noteheadnames", "Triangle Up") },
+      {"triangle-down",  QT_TRANSLATE_NOOP("noteheadnames", "Triangle Down") },
+      {"slashed1",       QT_TRANSLATE_NOOP("noteheadnames", "Slashed bottom left to top right") },
+      {"slashed2",       QT_TRANSLATE_NOOP("noteheadnames", "Slashed top left to bottom right") },
+      {"diamond",        QT_TRANSLATE_NOOP("noteheadnames", "Diamond") },
+      {"diamond-old",    QT_TRANSLATE_NOOP("noteheadnames", "Diamond (Old)") },
+      {"circled",        QT_TRANSLATE_NOOP("noteheadnames", "Circled") },
+      {"circled-large",  QT_TRANSLATE_NOOP("noteheadnames", "Circled") },
```

Comment:
Twice the same user visible and translatable text on different heads? Maybe better use `Circled (large)`?